core/src/main/java/com/es/core/model/order/Order.java

```diff
@@ -2,10 +2,11 @@
 
 import java.math.BigDecimal;
 import java.util.List;
+import java.util.UUID;
 
 public class Order
 {
-    private Long id;
+    private String id;
```

Comment:
better to have some "secureId" property, to order ids be more readable for managers, couriers etc.